modules/@apostrophecms/asset/index.js

```diff
@@ -243,6 +255,11 @@ module.exports = {
                 },
                 self.apos
               ));
+              if (fs.existsSync(cssPath)) {
+                fs.writeFileSync(cssPath, self.filterCss(fs.readFileSync(cssPath, 'utf8'), {
```

Comment:
By factoring the filterCss code this way it can be applied to both `ui/public/*.css` files and SCSS found in `ui/src` and `ui/apos` etc.